src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -91,10 +91,8 @@ protected MQTTBridge(Topics topics, TopicMapping topicMapping, MessageBridge mes
 
     @Override
     public void install() {
-        mappingConfigTopics =
-                this.config.lookupTopics(KernelConfigResolver.CONFIGURATION_CONFIG_KEY, MQTT_TOPIC_MAPPING);
-
-        mappingConfigTopics.subscribe((whatHappened, node) -> {
+        this.config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe((whatHappened, node) -> {
```

Comment:
ignore changes we don't care about. Many changes can happen and what happens inside the subscribe may be expensive to redo.

Reply:
That would be `timestampUpdated` and `interiorAdded`, correct?

Reply:
Yep

Reply:
Thanks!